src/librustc_mir/interpret/memory.rs

```diff
@@ -881,38 +908,23 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> Memory<'mir, 'tcx, M> {
             }
         }
 
-        // copy definedness to the destination
-        self.copy_undef_mask(src, dest, size, length)?;
+        // now copy over the undef data
```

Comment:
This doesn't copy any *data*, the comment is confusing.

Reply:
Rolled it back to the comment we had before.